src/test/kotlin/blackjack/CardCalculatorTest.kt

```diff
@@ -0,0 +1,48 @@
+package blackjack
+
+import io.kotest.matchers.shouldBe
+import org.junit.jupiter.api.Test
+
+class CardCalculatorTest {
+    @Test
+    fun `숫자 계산은 카드의 숫자를 기본으로 한다`() {
+        val cards = listOf(
+            Card.from(CardType.SPADE, CardValue.TWO),
+            Card.from(CardType.CLOVER, CardValue.THREE),
+            Card.from(CardType.DIAMOND, CardValue.SIX),
+            Card.from(CardType.HEART, CardValue.EIGHT),
+        )
+        CardCalculator.sum(cards) shouldBe 19
+    }
+
+    @Test
+    fun `J, Q, K는 10으로 계산한다`() {
+        val cards = listOf(
+            Card.from(CardType.SPADE, CardValue.TWO),
+            Card.from(CardType.CLOVER, CardValue.THREE),
+            Card.from(CardType.DIAMOND, CardValue.SIX),
+            Card.from(CardType.SPADE, CardValue.KING)
+        )
+        CardCalculator.sum(cards) shouldBe 21
+    }
+
+    @Test
+    fun `A는 1로 계산할 수 있다`() {
```

Comment:
```suggestion
    fun `A는 XXX 조건이면 1로 계산할 수 있다`() {
```
A는 어떤 경우에 1로 계산하고 또 11로 계산할 수 있는지 구체적으로 작성해보시면 어떨까요?